src/ensembl/src/content/app/entity-viewer/gene-view/components/base-pairs-ruler/BasePairsRuler.tsx

```diff
@@ -27,10 +27,14 @@ type Ticks = {
   labelledTicks: number[];
 };
 
+export type OnTicksCalculatedPayload = Ticks & {
```

Comment:
Wouldn't it be better to call this `TicksAndScale` instead of `OnTicksCalculatedPayload`? I feel the former is more definitive.

Reply:
✔️